src/style.css

```
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
```

font-weight is not working properly now
```suggestion
  font-family: Avenir;
  src: url("fonts/Avenir.ttc");
}

@font-face {
  font-family: Avenir;
  src: url("fonts/Avenir-Book.ttf");
  font-weight: 300;
}

@font-face {
  font-family: Avenir;
```